Publish script is broken

#### What project is affected?

Virtual Assistant

#### What language is this in?

C#

#### What happens?

The publish script is broken.

#### What are the steps to reproduce this issue?

Try to use the publish script from a random directory.

#### What were you expecting to happen?

I expected it to work.

#### Can you share any logs, error output, etc.?

```
PS C:\Users\ericc\source\repos\> dotnet publish -c release -o $publishFolder
Microsoft (R) Build Engine version 16.5.0+d4cbfca49 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.
```

#### Any screenshots or additional context?

The dotnet publish command requires a project or solution.

--- a/samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/Deployment/Scripts/publish.ps1
+++ b/samples/csharp/assistants/enterprise-assistant/VirtualAssistantSample/Deployment/Scripts/publish.ps1
@@ -50,14 +50,13 @@ if (-not $resourceGroup) {
     $resourceGroup = Read-Host "? Bot Resource Group"
 }
 
+# Get path to csproj file
+$projFile = Get-ChildItem $projFolder `
+    | Where-Object {$_.extension -eq ".csproj" } `
+    | Select-Object -First 1
+
 # Check for existing deployment files
 if (-not (Test-Path (Join-Path $projFolder '.deployment'))) {
-
-	# Get path to csproj file
-	$projFile = Get-ChildItem $projFolder `
-		| Where-Object {$_.extension -eq ".csproj" } `
-		| Select-Object -First 1
-
 	# Add needed deployment files for az
 	az bot prepare-deploy --lang Csharp --code-dir $projFolder --proj-file-path $projFile.name --output json | Out-Null
 }
@@ -70,7 +69,7 @@ if (Test-Path $zipPath) {
 
 # Perform dotnet publish step ahead of zipping up
 $publishFolder = $(Join-Path $projFolder 'bin\Release\netcoreapp3.0')
-dotnet publish -c release -o $publishFolder -v q > $logFile
+dotnet publish $projFile -c release -o $publishFolder -v q > $logFile
 
 if($?) {
     # Compress source code

--- a/samples/csharp/assistants/hospitality-assistant/VirtualAssistantSample/Deployment/Scripts/publish.ps1
+++ b/samples/csharp/assistants/hospitality-assistant/VirtualAssistantSample/Deployment/Scripts/publish.ps1
@@ -50,14 +50,13 @@ if (-not $resourceGroup) {
     $resourceGroup = Read-Host "? Bot Resource Group"
 }
 
+# Get path to csproj file
+$projFile = Get-ChildItem $projFolder `
+    | Where-Object {$_.extension -eq ".csproj" } `
+    | Select-Object -First 1
+
 # Check for existing deployment files
 if (-not (Test-Path (Join-Path $projFolder '.deployment'))) {
-
-	# Get path to csproj file
-	$projFile = Get-ChildItem $projFolder `
-		| Where-Object {$_.extension -eq ".csproj" } `
-		| Select-Object -First 1
-
 	# Add needed deployment files for az
 	az bot prepare-deploy --lang Csharp --code-dir $projFolder --proj-file-path $projFile.name --output json | Out-Null
 }
@@ -70,7 +69,7 @@ if (Test-Path $zipPath) {
 
 # Perform dotnet publish step ahead of zipping up
 $publishFolder = $(Join-Path $projFolder 'bin\Release\netcoreapp3.0')
-dotnet publish -c release -o $publishFolder -v q > $logFile
+dotnet publish $projFile -c release -o $publishFolder -v q > $logFile
 
 if($?) {
     # Compress source code

--- a/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/Deployment/Scripts/publish.ps1
+++ b/samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/Deployment/Scripts/publish.ps1
@@ -50,14 +50,13 @@ if (-not $resourceGroup) {
     $resourceGroup = Read-Host "? Bot Resource Group"
 }
 
+# Get path to csproj file
+$projFile = Get-ChildItem $projFolder `
+    | Where-Object {$_.extension -eq ".csproj" } `
+    | Select-Object -First 1
+
 # Check for existing deployment files
 if (-not (Test-Path (Join-Path $projFolder '.deployment'))) {
-
-	# Get path to csproj file
-	$projFile = Get-ChildItem $projFolder `
-		| Where-Object {$_.extension -eq ".csproj" } `
-		| Select-Object -First 1
-
 	# Add needed deployment files for az
 	az bot prepare-deploy --lang Csharp --code-dir $projFolder --proj-file-path $projFile.name --output json | Out-Null
 }
@@ -70,7 +69,7 @@ if (Test-Path $zipPath) {
 
 # Perform dotnet publish step ahead of zipping up
 $publishFolder = $(Join-Path $projFolder 'bin\release\netcoreapp3.0')
-dotnet publish -c release -o $publishFolder -v q > $logFile
+dotnet publish $projFile -c release -o $publishFolder -v q > $logFile
 
 if($?) {
     # Compress source code

